server/controllers/admin.py

```diff
@@ -637,9 +637,19 @@ def enrollment(cid):
     return render_template('staff/course/enrollment.html',
                            enrollments=students, staff=staff,
                            single_form=single_form,
+                           unenroll_form=forms.CSRFForm(),
                            courses=courses,
                            current_course=current_course)
 
+@admin.route("/course/<int:cid>/<string:user_id>/unenroll", methods=['POST'])
```

Comment:
I though it made sense to have the user_id be a part of the route from an API standpoint. However, if you think it really should be a part of the form data, I'd be happy to change it. Let me know 👍 

Also changed user_id to an integer.